spec/services/anonymized_intake_csv_service_spec.rb

```diff
@@ -45,6 +45,127 @@
         end
       end
     end
+
+    context "intake status columns" do
+      before do
+        create(
+          :ticket_status,
+          intake: intake_1,
+          created_at: DateTime.new(2020, 1, 1),
+          intake_status: EitcZendeskInstance::INTAKE_STATUS_GATHERING_DOCUMENTS,
+        )
+        create(
+          :ticket_status,
+          intake: intake_1,
+          created_at: DateTime.new(2020, 1, 2),
+          intake_status: EitcZendeskInstance::INTAKE_STATUS_IN_REVIEW,
+        )
+        create(
+          :ticket_status,
+          intake: intake_2,
+          verified_change: false,
+          created_at: DateTime.new(2020, 1, 3),
+          intake_status: EitcZendeskInstance::INTAKE_STATUS_GATHERING_DOCUMENTS,
+        )
+        create(
+          :ticket_status,
+          intake: intake_2,
+          created_at: DateTime.new(2020, 1, 4),
+          intake_status: EitcZendeskInstance::INTAKE_STATUS_IN_REVIEW,
+        )
+      end
+
+      it "stores timestamps in the appropriate column" do
+        row_1_csv_data = csv[0]
+        expect(row_1_csv_data["Intake Status - Gathering Documents"]).to eq("2020-01-01 00:00:00 UTC")
+        expect(row_1_csv_data["Intake Status - In Review"]).to eq("2020-01-02 00:00:00 UTC")
+        row_2_csv_data = csv[1]
+        expect(row_2_csv_data["Intake Status - Gathering Documents"]).to be_nil
```

Comment:
it seems that in my previous comment i was wrong but i can't quite tell why...

Reply:
The answer is in `verified_change: false` on line 67

https://github.com/codeforamerica/vita-min/pull/441/files#diff-9b3aa8c374ae5eda5bb4a7e87050bb17R67

Ben told me this is important, but I can't quite figure out why, so I'm going to retain the behavior and document in a follow-up commit.